server/package.json

```diff
@@ -28,6 +29,7 @@
     "express": "4.17.1",
     "fast-csv": "4.3.6",
     "fast-xml-parser": "3.17.4",
+    "handlebars": "4.7.7",
```

Comment:
I think the last decision was plaintext emails so handlebars may not be needed

Reply:
Yep plaintext is completely fine, assuming we can support some basic HTML like the mailto link and bolded text

Reply:
For reference, this is what the email looks like on my phone. 
Even with just html, I think we can make it look a bit nicer. 

![Screenshot_20210518-143143](https://user-images.githubusercontent.com/54554766/118726311-fcb44b00-b7e5-11eb-9411-dbf9513f9444.png)

Reply:
We could but it's also not a priority - I think first let's get to the point we can send a sample/test email over to the program area, we can adjust this if we have time 😄 The BC Vaccination email is even less fancy than this (no buttons)